### PR TITLE
Fix: Workflows issue

### PR DESCRIPTION
**Pull request by gaurav-jangam**
###Description
- Workflow status not updating.
- Deleted travis.yml file and added ci.yml file in .github/workflows folder.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.PRIVATE_ACCESS_TOKEN }}
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v2.5.1
+      with:
+        node-version: 12.x
+        cache: 'npm'
```
Comment (Nikschavan) [inline .github/workflows/ci.yml]: We don't need npm, this can be removed

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    - name: Set composer token
+      run: composer config github-oauth.github.com ${{ secrets.PRIVATE_ACCESS_TOKEN }} 
+
+    - name: Composer Install
+      uses: ramsey/composer-install@v1
+
+    - name: NPM Dependencies
+      if: always()
+      run: npm install
```
Comment (Nikschavan) [inline .github/workflows/ci.yml]: This can be removed since we don't have npm dependencies

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    - name: Commit ESLint fixes
+      if: always()
+      uses: EndBug/add-and-commit@v7
+      with:
+          add: 'inc/lib/onboarding/*.js'
+          default_author: github_actor
+          message: 'Commit ESLint auto fixes'
+          pathspec_error_handling: exitAtEnd
+          push: true
```
Comment (Nikschavan) [inline .github/workflows/ci.yml]: This can be removed

File: .github/workflows/ci.yml (hunk 1 above)
Comment (gaurav-jangam) [reply .github/workflows/ci.yml]: Done

File: .github/workflows/ci.yml (hunk 2 above)
Comment (gaurav-jangam) [reply .github/workflows/ci.yml]: Done

File: .github/workflows/ci.yml (hunk 3 above)
Comment (gaurav-jangam) [reply .github/workflows/ci.yml]: Done

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+concurrency:
+    # The concurrency group contains the workflow name and the branch name for pull requests
+    # or the commit hash for any other events.
+    group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+  lint-and-audit:
+
+    runs-on: self-hosted
```
Comment (Nikschavan) [inline .github/workflows/ci.yml]:
```suggestion
    runs-on: ubuntu-latest
```

Comment (gaurav-jangam) [reply .github/workflows/ci.yml]: Done

**Review by Nikschavan — APPROVED** [review]

**Pull request merged**